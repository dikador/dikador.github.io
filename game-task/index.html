<!DOCTYPE html><html lang=""><head><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1"><link rel="icon" href="/game-task/favicon.ico"><title>game-task</title><link href="/game-task/css/app.982205c8.css" rel="preload" as="style"><link href="/game-task/js/app.030119bd.js" rel="preload" as="script"><link href="/game-task/js/chunk-vendors.dc5372e2.js" rel="preload" as="script"><link href="/game-task/css/app.982205c8.css" rel="stylesheet"></head><body><noscript><strong>We're sorry but game-task doesn't work properly without JavaScript enabled. Please enable it to continue.</strong></noscript><div id="app"></div><script src="/game-task/js/chunk-vendors.dc5372e2.js"></script><script src="/game-task/js/app.030119bd.js"></script></body></html>