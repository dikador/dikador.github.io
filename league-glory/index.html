<!DOCTYPE html><html lang="ru"><head>    <meta charset="UTF-8">    <meta name="viewport" content="width=device-width, initial-scale=1.0">    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"        integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">    <link rel="preconnect" href="https://fonts.googleapis.com">    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">    <link rel="stylesheet" href="css/style.min.css">    <title>Document</title></head><body>    <header class="header">        <div class="container">            <div class="header__body d-flex justify-content-between align-items-start">                <a href="index.html" class="header__logo">                    <picture><source srcset="img/logo.svg" type="image/webp"><img src="img/logo.svg" alt=""></picture>                </a>                <nav class="px-4 w-md-auto w-100 d-flex flex-column">                    <ul class="d-flex mb-0 p-0 w-100 justify-content-between header__nav">                        <li class="nav-item "><a href="sportsmens.html" class="nav-link">Спортсмены</a></li>                        <li class="nav-item"><a href="schols.html" class="nav-link">Спортивные школы</a></li>                        <li class="nav-item "><a href="events.html" class="nav-link">Организаторы мероприятий</a></li>                        <li class="nav-item"><a href="objects.html" class="nav-link">Спортивные объекты</a></li>                    </ul>                    <div class="header__search_block d-flex justify-content-center w-100 ">                        <input placeholder="Я ищу" type="text" class="header__search main-search">                    </div>                </nav>                <div class="header__geo nav-item d-flex">                    <picture><source srcset="img/geo.webp" type="image/webp"><img src="img/geo.png" alt=""></picture>                    <p class="p-0 ms-2">Москва</p>                </div>                <div class="header__burger">                    <span></span>                </div>            </div>        </div>    </header>    <main>        <section class="intro">            <div class="container">                <div class="intro__body">                    <div class="intro__text">                        <h3>СПОРТИВНЫЙ СЕРВИС</h3>                        <h1>ЛИГА СЛАВЫ</h1>                        <h2>ТВОЙ путь к успеху в спорте</h2>                    </div>                </div>            </div>        </section>        <section class="opportunities">            <div class="container">                <h2 class="main-title pb-md-0 pb-2 mb-0">мы даем возможность</h2>                <div class="mt-md-5 mt-4 row">                    <div class="opportunities__block d-flex col-md-4 col-12 mb-md-0 mb-2 align-items-center">                        <h3 class="opportunities__block-title me-md-5 me-4 mb-0">1</h3>                        <p class="opporunities__block-text">                            Найти спортивную школу,                            подходящую именно Вам                        </p>                    </div>                    <div class="opportunities__block d-flex col-md-4 col-12 mb-md-0 mb-2 align-items-center">                        <h3 class="opportunities__block-title me-md-3 me-4 mb-0">2</h3>                        <p class="opporunities__block-text">                            Участвовать в соревнованиях разного уровня                        </p>                    </div>                    <div class="opportunities__block d-flex col-md-4 col-12 mb-md-0 align-items-center">                        <h3 class="opportunities__block-title me-md-3 me-4 mb-0">3</h3>                        <p class="opporunities__block-text pe-lg-3">                            Заниматься и проводить соревнования на самых лучших спортивных площадках                        </p>                    </div>                </div>            </div>        </section>        <section class="services">            <div class="container">                <h2 class="main-title pb-3 mb-0">для кого подойдет сервис</h2>                <div class="services__card row text-left mt-md-5 mt-4 pb-md-0 pb-3">                    <div class="col-md-6 col-12">                        <picture><source srcset="img/mainBlock-1.webp" type="image/webp"><img class="me-4 w-100" src="img/mainBlock-1.jpg" alt=""></picture>                    </div>                    <div class="col-md-6 col-12 services__card-body d-flex flex-column justify-content-between">                        <div>                            <h4 class="services__card-title mt-md-0 mt-3 mb-0">Организаторам соревнований</h4>                            <p class="services__desc main-card-desc mt-lg-4 mt-3">Органищаторы соревнований могут найти спортивые площадки                                для                                проведения                                соревнований, найти на сервисе спортсменов, спортивные школы для участия в соревнованиях</p>                        </div>                        <button type="button" class="main-card-button services__button">НАЙТИ ОРГАНИЗАТОРА</button>                    </div>                </div>                <div class="services__card row text-left mt-md-5 mt-4 pb-md-0 pb-3">                    <div class="col-md-6 col-12">                        <picture><source srcset="img/mainBlock-2.webp" type="image/webp"><img class="me-4 w-100" src="img/mainBlock-2.jpg" alt=""></picture>                    </div>                    <div class="col-md-6 col-12 services__card-body d-flex flex-column justify-content-between">                        <div>                            <h4 class="services__card-title mt-md-0 mt-3 mb-0">Спортивным школам</h4>                            <p class="services__desc main-card-desc mt-lg-4 mt-3">Организаторы соревнований могут найти спортивые площадки                                для                                проведения                                соревнований, найти на сервисе спортсменов, спортивные школы для участия в соревнованиях</p>                        </div>                        <button type="button" class="main-card-button services__button">НАЙТИ ШКОЛУ</button>                    </div>                </div>                <div class="services__card row text-left mt-md-5 mt-4 pb-md-0 pb-3">                    <div class="col-md-6 col-12">                        <picture><source srcset="img/mainBlock-3.webp" type="image/webp"><img class="me-4 w-100" src="img/mainBlock-3.jpg" alt=""></picture>                    </div>                    <div class="col-md-6 col-12 services__card-body d-flex flex-column justify-content-between">                        <div>                            <h4 class="services__card-title mt-md-0 mt-3 mb-0">Спортсменам</h4>                            <p class="services__desc main-card-desc mt-lg-4 mt-3">Организаторы соревнований могут найти спортивые площадки                                для                                проведения                                соревнований, найти на сервисе спортсменов, спортивные школы для участия в соревнованиях</p>                        </div>                        <button type="button" class="main-card-button services__button">НАЙТИ ШКОЛУ</button>                    </div>                </div>                <div class="services__card row text-left mt-md-5 mt-4 pb-md-0 pb-3">                    <div class="col-md-6 col-12">                        <picture><source srcset="img/mainBlock-4.webp" type="image/webp"><img class="me-4 w-100" src="img/mainBlock-4.jpg" alt=""></picture>                    </div>                    <div class="col-md-6 col-12 services__card-body d-flex flex-column justify-content-between">                        <div>                            <h4 class="services__card-title mt-md-0 mt-3 mb-0">Владельцам спортивных объектов</h4>                            <p class="services__desc main-card-desc mt-lg-4 mt-3">Организаторы соревнований могут найти спортивые площадки                                для                                проведения                                соревнований, найти на сервисе спортсменов, спортивные школы для участия в соревнованиях</p>                        </div>                        <button type="button" class="main-card-button services__button">НАЙТИ ОБЪЕКТ</button>                    </div>                </div>            </div>        </section>        <section class="events">            <div class="container">                <h2 class="main-title pb-2">Ближайшие соревнования</h2>                <div id="carouselExampleIndicators" class="carousel slide" data-bs-ride="carousel">                    <div class="carousel-indicators">                        <button type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide-to="0" class="active"                            aria-current="true" aria-label="Slide 1"></button>                        <button type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide-to="1"                            aria-label="Slide 2"></button>                        <button type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide-to="2"                            aria-label="Slide 3"></button>                    </div>                    <div class="row mt-sm-5 mt-4 pb-4 pb-sm-3 carousel-inner mx-0">                        <div class="events__item col-sm-4 col-12 text-start active ">                            <picture><source srcset="img/imagemainBlock-5.webp" type="image/webp"><img class="w-100" src="img/imagemainBlock-5.jpg" alt=""></picture>                            <div class="mt-3">                                <h4 class="main-card-desc mb-2 fw-500">ПЕРВЕНСТВО ПО ФУТБОЛУ                                    СРЕДИ ДЕТЕЙ 2009 ГОДА РОЖДЕНИЯ</h4>                                <p class="main-card-desc pt-1 mb-4">Спорткомплекс «Кедр»</p>                                <div class="d-flex justify-content-between align-items-center">                                    <p class="main-card-desc">21.09.2021 г.</p>                                    <div>                                        <picture><source srcset="img/like.svg" type="image/webp"><img class="card-events me-md-4 me-sm-2 me-4" src="img/like.svg" alt=""></picture>                                        <picture><source srcset="img/share.webp" type="image/webp"><img class="card-events" src="img/share.png" alt=""></picture>                                    </div>                                </div>                            </div>                        </div>                        <div class="events__item col-sm-4 col-12 text-start ">                            <picture><source srcset="img/imagemainBlock-5.webp" type="image/webp"><img class="w-100" src="img/imagemainBlock-5.jpg" alt=""></picture>                            <div class="mt-3">                                <h4 class="main-card-desc mb-2 fw-500">ПЕРВЕНСТВО ПО ФУТБОЛУ                                    СРЕДИ ДЕТЕЙ 2019 ГОДА РОЖДЕНИЯ</h4>                                <p class="main-card-desc pt-1 mb-4">Спорткомплекс «Кедр»</p>                                <div class="d-flex justify-content-between align-items-center">                                    <p class="main-card-desc ">21.09.2021 г.</p>                                    <div>                                        <picture><source srcset="img/like.svg" type="image/webp"><img class="card-events me-md-4 me-sm-2 me-4" src="img/like.svg" alt=""></picture>                                        <picture><source srcset="img/share.webp" type="image/webp"><img class="card-events" src="img/share.png" alt=""></picture>                                    </div>                                </div>                            </div>                        </div>                        <div class="events__item col-sm-4 col-12 text-start ">                            <picture><source srcset="img/imagemainBlock-5.webp" type="image/webp"><img class="w-100" src="img/imagemainBlock-5.jpg" alt=""></picture>                            <div class="mt-3">                                <h4 class="main-card-desc mb-2 fw-500">ПЕРВЕНСТВО ПО ФУТБОЛУ                                    СРЕДИ ДЕТЕЙ 2000 ГОДА РОЖДЕНИЯ</h4>                                <p class="main-card-desc pt-1 mb-4">Спорткомплекс «Кедр»</p>                                <div class="d-flex justify-content-between align-items-center">                                    <p class="main-card-desc ">21.09.2021 г.</p>                                    <div>                                        <picture><source srcset="img/like.svg" type="image/webp"><img class="card-events me-md-4 me-sm-2 me-4" src="img/like.svg" alt=""></picture>                                        <picture><source srcset="img/share.webp" type="image/webp"><img class="card-events" src="img/share.png" alt=""></picture>                                    </div>                                </div>                            </div>                        </div>                    </div>                </div>                <div class="mt-2 text-center d-block">                    <button class="btn-text pt-sm-5 pt-4" type="button">ВСЕ СОРЕВНОВАНИЯ</button>                </div>            </div>        </section>    </main>    <footer class="footer">        <div class="container">            <div class="row justify-content-between">                <a class="footer__logo-adaptive col-4" href="#"><picture><source srcset="img/logo.svg" type="image/webp"><img src="img/logo.svg" alt=""></picture></a>                <ul class="col-lg-3 col-sm-4 col-12 mb-0 pb-sm-0 pb-2">                    <li class="nav-item"><a href="#" class="nav-link">Спортсмены</a></li>                    <li class="nav-item"><a href="#" class="nav-link">Спортивные школы</a></li>                    <li class="nav-item"><a href="#" class="nav-link">Организаторы мероприятий</a></li>                    <li class="nav-item"><a href="#" class="nav-link">Спортивные объекты</a></li>                </ul>                <a class="footer__logo mb-0 col-lg-4 col-sm-3 col-12" href="#"><picture><source srcset="img/logo.svg" type="image/webp"><img src="img/logo.svg" alt=""></picture></a>                <ul class="col-lg-3 col-sm-4 col-12 mb-0 mt-sm-0 mt-5">                    <li class="nav-item"><a href="#" class="nav-link">Политика конфиденциальности</a></li>                    <li class="nav-item"><a href="#" class="nav-link">Правила оферты</a></li>                    <li class="nav-item"><a href="#" class="nav-link">FAQ</a></li>                </ul>            </div>            <div class="footer__bot d-flex justify-content-between flex-sm-row flex-column">                <p class="main-card-desc text-white pb-sm-0 pb-2">© 2021 ООО «Лига славы»</p>                <p class="main-card-desc text-white mt-sm-0 mt-4">Условия использования © Gazprom Neft PJSC</p>            </div>        </div>    </footer>    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.10.2/dist/umd/popper.min.js"        integrity="sha384-7+zCNj/IqJ95wo16oMtfsKbZ9ccEh31eOz1HGyDuCQ6wgnyJNSYdrPa03rtR1zdB" crossorigin="anonymous"></script>    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.min.js"        integrity="sha384-QJHtvGhmr9XOIpI6YVutG+2QOK9T+ZnN4kzFN1RtK3zEFEIsxhlmWl5/YESvpZ13" crossorigin="anonymous"></script>    <script src="js/script.js"></script></body></html><!-- <svg class="a">    <use xlink:href="/img/symbol/sprite.svg#clock"></use></svg> -->