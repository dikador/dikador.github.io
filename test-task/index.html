<!DOCTYPE html><html lang=""><head><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1"><link rel="icon" href="/slider-task/favicon.ico"><title>slider-task</title><link href="/slider-task/css/app.52484dfb.css" rel="preload" as="style"><link href="/slider-task/css/chunk-vendors.638fc61b.css" rel="preload" as="style"><link href="/slider-task/js/app.1d0783dd.js" rel="preload" as="script"><link href="/slider-task/js/chunk-vendors.6e4a9451.js" rel="preload" as="script"><link href="/slider-task/css/chunk-vendors.638fc61b.css" rel="stylesheet"><link href="/slider-task/css/app.52484dfb.css" rel="stylesheet"></head><body><noscript><strong>We're sorry but slider-task doesn't work properly without JavaScript enabled. Please enable it to continue.</strong></noscript><div id="app"></div><script src="/slider-task/js/chunk-vendors.6e4a9451.js"></script><script src="/slider-task/js/app.1d0783dd.js"></script></body></html>