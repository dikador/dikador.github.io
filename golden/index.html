<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="preconnect" href="https://fonts.gstatic.com">
    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700;800&family=Roboto+Slab&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="css/style.min.css">
    <title>Golden</title>
</head>

<body>
    <header class="header">
        <div class="container">
            <div class="header__inner">
                <div class="header__logo">
                    <a href="">Golden</a>
                </div>
                <nav>
                    <ul class="nav">
                        <li class="nav-link">
                            <a class="nav-link" href="#">Home</a>
                        </li>
                        <li class="nav-link">
                            <a class="nav-link" href="#services">Services</a>
                        </li>
                        <li class="nav-link">
                            <a class="nav-link" href="#portfolio">Portfolio</a>
                        </li>
                        <li class="nav-link">
                            <a class="nav-link" href="#about">About</a>
                        </li>
                        <li class="nav-link">
                            <a class="nav-link" href="#contact">Contact</a>
                        </li>
                    </ul>
                </nav>

                <div onclick="openBurger()" class="header__burger">
                    <span></span>
                </div>
            </div>
        </div>
    </header>
    <main>

        <div class="wrapper">
            <section class="intro">
                <div class="intro__inner">
                    <h2 class="intro__sub-title">Welcome To Our Studio!</h2>
                    <h1 class="intro__main-title">Itâ€™s nice to Meet you</h1>
                    <a href="#contact" class="button btn-intro btn-yelow">Tell Me More</a>
                </div>
            </section>
        </div>



        <section id="services" class="services">
            <div class="container">
                <div class="title__page">
                    <h2 class="title">Services</h2>
                    <h3 class="sub__title">Proin iaculis purus consequat sem cure. </h3>
                </div>

                <div class="services__inner">
                    <div class="services__block">
                        <picture>
                            <source srcset="img/serv-1.svg" type="image/webp"><img src="img/serv-1.svg" alt="basket">
                        </picture>
                        <h4>E-Commerce</h4>
                        <p>Proin iaculis purus consequat sem cure
                            digni ssim. Donec porttitora entum suscipit
                            aenean rhoncus posuere odio in tincidunt.</p>
                    </div>

                    <div class="services__block">
                        <picture>
                            <source srcset="img/serv-2.svg" type="image/webp"><img src="img/serv-2.svg" alt="notebook">
                        </picture>
                        <h4>Responsive Web</h4>
                        <p>Proin iaculis purus consequat sem cure
                            digni ssim. Donec porttitora entum suscipit
                            aenean rhoncus posuere odio in tincidunt.</p>
                    </div>

                    <div class="services__block">
                        <picture>
                            <source srcset="img/serv-3.svg" type="image/webp"><img src="img/serv-3.svg" alt="lock">
                        </picture>
                        <h4>Web Security</h4>
                        <p>Proin iaculis purus consequat sem cure
                            digni ssim. Donec porttitora entum suscipit
                            aenean rhoncus posuere odio in tincidunt.</p>
                    </div>
                </div>
            </div>
        </section>


        <section id="portfolio" class="portfolio">
            <div class="container">
                <div class="title__page">
                    <h2 class="title">Our portfolio</h2>
                    <h3 class="sub__title">Proin iaculis purus consequat sem cure.</h3>
                </div>
                <div class="portfolio__inner">
                    <div class="portfolio__block">
                        <div class="portfolio__block__img">
                            <picture>
                                <source srcset="img/portf-1.png" type="image/webp"><img src="img/portf-1.png" alt="">
                            </picture>

                            <div class="portfolio__stels">
                                <a href="">
                                    <h5 class="portfolio__stels__title">Lorem ipsum</h5>
                                </a>
                            </div>
                        </div>


                        <div class="portfolio__block__item">
                            <a href="">
                                <h4 class="title__block portfolio__title-block">Ebony & Ivory</h4>
                            </a>
                            <h5 class="sub__title block__sub-title">Branding</h5>
                        </div>

                    </div>

                    <div class="portfolio__block">
                        <div class="portfolio__block__img">
                            <picture>
                                <source srcset="img/portf-2.webp" type="image/webp"><img src="img/portf-2.svg" alt="">
                            </picture>

                            <div class="portfolio__stels">
                                <a href="">
                                    <h5 class="portfolio__stels__title">Lorem ipsum</h5>
                                </a>
                            </div>
                        </div>
                        <div class="portfolio__block__item">
                            <a href="">
                                <h4 class="title__block portfolio__title-block ">Smart Stationary</h4>
                            </a>
                            <h5 class="sub__title block__sub-title">Print Design</h5>
                        </div>
                    </div>
                    <div class="portfolio__block">
                        <div class="portfolio__block__img">
                            <picture>
                                <source srcset="img/portf-3.webp" type="image/webp"><img src="img/portf-3.svg" alt="">
                            </picture>

                            <div class="portfolio__stels">
                                <a href="">
                                    <h5 class="portfolio__stels__title">Lorem ipsum</h5>
                                </a>
                            </div>
                        </div>
                        <div class="portfolio__block__item">
                            <a href="">
                                <h4 class="title__block portfolio__title-block">Clever Poster</h4>
                            </a>
                            <h5 class="sub__title block__sub-title">Print Design</h5>
                        </div>
                    </div>

                    <div class="portfolio__block">
                        <div class="portfolio__block__img">
                            <picture>
                                <source srcset="img/portf-4.webp" type="image/webp"><img src="img/portf-4.svg" alt="">
                            </picture>

                            <div class="portfolio__stels">
                                <a href="">
                                    <h5 class="portfolio__stels__title">Lorem ipsum</h5>
                                </a>
                            </div>
                        </div>

                        <div class="portfolio__block__item">
                            <a href="">
                                <h4 class="title__block portfolio__title-block">Vinyl Record</h4>
                            </a>
                            <h5 class="sub__title block__sub-title">Product Mock-Up</h5>
                        </div>
                    </div>



                    <div class="portfolio__block">
                        <div class="portfolio__block__img">
                            <picture>
                                <source srcset="img/portf-5.jpg" type="image/webp"><img src="img/portf-5.jpg" alt="">
                            </picture>
                            <div class="portfolio__stels">
                                <a href="">
                                    <h5 class="portfolio__stels__title">Lorem ipsum</h5>
                                </a>
                            </div>
                        </div>
                        <div class="portfolio__block__item">
                            <a href="">
                                <h4 class="title__block portfolio__title-block">Treehouse Template
                                </h4>
                            </a>
                            <h5 class="sub__title block__sub-title">Web Design</h5>
                        </div>
                    </div>

                    <div class="portfolio__block">
                        <div class="portfolio__block__img">
                            <picture>
                                <source srcset="img/portf-6.webp" type="image/webp"><img src="img/portf-6.png" alt="">
                            </picture>

                            <div class="portfolio__stels">
                                <a href="">
                                    <h5 class="portfolio__stels__title">Lorem ipsum</h5>
                                </a>
                            </div>
                        </div>
                        <div class="portfolio__block__item">
                            <a href="">
                                <h4 class="title__block portfolio__title-block">Burned Logo</h4>
                            </a>
                            <h5 class="sub__title block__sub-title">Branding</h5>
                        </div>
                    </div>
                </div>
            </div>
        </section>


        <section id="about" class="about">
            <div class="container">
                <div class="title__page">
                    <h2 class="title">About Us</h2>
                    <h3 class="sub__title">Proin iaculis purus consequat sem cure. </h3>
                </div>

                <div class="about__inner">



                    <div class="about__block">
                        <div class="about__block__text">
                            <h4 class="title__block about__title"><span>July 2010</span> <br>
                                Our Humble Beginnings
                            </h4>
                            <p>Proin iaculis purus consequat sem cure
                                digni ssim. Donec porttitora entum suscipit
                                aenean rhoncus posuere odio in tincidunt. Proin
                                iaculis purus consequat sem cure digni
                                ssim. Donec porttitora entum suscipit.</p>
                        </div>


                        <div class="about__block__imeage">
                            <picture>
                                <source srcset="img/about-1.jpg" type="image/webp"><img src="img/about-1.jpg" alt="things on the table">
                            </picture>
                        </div>
                    </div>



                    <div class="about__block">
                        <div class="about__block__text text-reverse">
                            <h4 class="title__block about__title"><span>January 2011</span> <br>
                                Facing Startup Battles

                            </h4>
                            <p>Proin iaculis purus consequat sem cure
                                digni ssim. Donec porttitora entum suscipit
                                aenean rhoncus posuere odio in tincidunt. Proin
                                iaculis purus consequat sem cure digni
                                ssim. Donec porttitora entum suscipit.</p>
                        </div>


                        <div class="about__block__imeage">
                            <img src="img/about-2.jpg" alt="things on the table">
                        </div>
                    </div>



                    <div class="about__block">
                        <div class="about__block__text">
                            <h4 class="title__block about__title"><span>December 2012</span> <br>
                                Enter The Dark Days


                            </h4>
                            <p>Proin iaculis purus consequat sem cure
                                digni ssim. Donec porttitora entum suscipit
                                aenean rhoncus posuere odio in tincidunt. Proin
                                iaculis purus consequat sem cure digni
                                ssim. Donec porttitora entum suscipit.</p>
                        </div>


                        <div class="about__block__imeage">
                            <picture>
                                <source srcset="img/about-3.webp" type="image/webp"><img src="img/about-3.jpg" alt="things on the table">
                            </picture>
                        </div>
                    </div>



                    <div class="about__block">
                        <div class="about__block__text text-reverse">
                            <h4 class="title__block about__title"><span>february 2014</span> <br>
                                Our Triumph


                            </h4>
                            <p>Proin iaculis purus consequat sem cure
                                digni ssim. Donec porttitora entum suscipit
                                aenean rhoncus posuere odio in tincidunt. Proin
                                iaculis purus consequat sem cure digni
                                ssim. Donec porttitora entum suscipit.</p>
                        </div>


                        <div class="about__block__imeage">
                            <picture>
                                <source srcset="img/about-4.webp" type="image/webp"><img src="img/about-4.jpg" alt="things on the table">
                            </picture>
                        </div>
                    </div>

                    <div class="about__buttons">
                        <div class="about__block__imeage about__block__button">
                            <button onclick="moreHistory()" type="button" class="button btn-yelow btn-circle">Our Story
                                Continues ...</button>
                        </div>
                    </div>


                    <div class="about__more-block">

                        <div class="about__block">
                            <div class="about__block__text">
                                <h4 class="title__block about__title"><span>July 2015</span> <br>
                                    Our Humble Beginnings
                                </h4>
                                <p>Proin iaculis purus consequat sem cure
                                    digni ssim. Donec porttitora entum suscipit
                                    aenean rhoncus posuere odio in tincidunt. Proin
                                    iaculis purus consequat sem cure digni
                                    ssim. Donec porttitora entum suscipit.</p>
                            </div>


                            <div class="about__block__imeage">
                                <picture>
                                    <source srcset="img/about-1.jpg" type="image/webp"><img src="img/about-1.jpg" alt="things on the table">
                                </picture>
                            </div>
                        </div>



                        <div class="about__block">
                            <div class="about__block__text text-reverse">
                                <h4 class="title__block about__title"><span>february 2016</span> <br>
                                    Our Triumph


                                </h4>
                                <p>Proin iaculis purus consequat sem cure
                                    digni ssim. Donec porttitora entum suscipit
                                    aenean rhoncus posuere odio in tincidunt. Proin
                                    iaculis purus consequat sem cure digni
                                    ssim. Donec porttitora entum suscipit.</p>
                            </div>


                            <div class="about__block__imeage about__block-end">
                                <picture>
                                    <source srcset="img/about-4.webp" type="image/webp"><img src="img/about-4.jpg" alt="things on the table">
                                </picture>
                            </div>
                        </div>

                    </div>




                </div>
            </div>
        </section>

        <section class="team">
            <div class="container">
                <div class="title__page">
                    <h2 class="title">our amazing team</h2>
                    <h3 class="sub__title">Proin iaculis purus consequat sem </h3>
                </div>
                <div class="team__inner">
                    <div class="team__block">
                        <div class="team__block-imeage">
                            <a href="">
                                <picture>
                                    <source srcset="img/businessman.svg" type="image/webp"><img src="img/businessman.svg" alt="avatar man">
                                </picture>
                            </a>
                        </div>
                        <div class="team__block-title">
                            <a href="">
                                <h4 class="title__block block-team-title">Kimberly Thompson</h4>
                            </a>
                            <p>Marketer</p>
                        </div>

                        <div class="social">
                            <a class="twitter-social" href="#"><i class="fa social-block twitter-block  fa-twitter" aria-hidden="true"></i></a>

                            <a class="facebook-social" href="#"><i class="fa social-block facebook-block fa-facebook" aria-hidden="true"></i></a>

                            <a class="google-social" href="#"><i class="fa social-block google-block fa-google-plus" aria-hidden="true"></i></a>
                        </div>
                    </div>

                    <div class="team__block">
                        <div class="team__block-imeage">
                            <a href="">
                                <picture>
                                    <source srcset="img/businessman.svg" type="image/webp"><img src="img/businessman.svg" alt="avatar man">
                                </picture>
                            </a>
                        </div>
                        <div class="team__block-title">
                            <a href="">
                                <h4 class="title__block block-team-title">Alex Thompson</h4>
                            </a>
                            <p>coder</p>
                        </div>

                        <div class="social">
                            <a class="twitter-social" href="#"><i class="fa social-block twitter-block  fa-twitter" aria-hidden="true"></i></a>

                            <a class="facebook-social" href="#"><i class="fa social-block facebook-block fa-facebook" aria-hidden="true"></i></a>

                            <a class="google-social" href="#"><i class="fa social-block google-block fa-google-plus" aria-hidden="true"></i></a>
                        </div>
                    </div>

                    <div class="team__block">
                        <div class="team__block-imeage">
                            <a href="">
                                <picture>
                                    <source srcset="img/boss.svg" type="image/webp"><img src="img/boss.svg" alt="avatar man">
                                </picture>
                            </a>
                        </div>
                        <div class="team__block-title">
                            <a href="">
                                <h4 class="title__block block-team-title">Willson barker</h4>
                            </a>
                            <p>Graphic Designer</p>
                        </div>

                        <div class="social">
                            <a class="twitter-social" href="#"><i class="fa social-block twitter-block  fa-twitter" aria-hidden="true"></i></a>

                            <a class="facebook-social" href="#"><i class="fa social-block facebook-block fa-facebook" aria-hidden="true"></i></a>

                            <a class="google-social" href="#"><i class="fa social-block google-block fa-google-plus" aria-hidden="true"></i></a>
                        </div>
                    </div>
                </div>
                <p class="team__text">Proin iaculis purus consequat sem cure digni ssim donec porttitora entum suscipit
                    aenean rhoncus posuere odio in tincidunt proin iaculis.</p>
            </div>
        </section>

        <div class="brands">
            <div class="container">
                <div class="brands__wrapper">
                    <div class="brand__block">
                        <a href="">
                            <picture>
                                <source srcset="img/brands-1.png" type="image/webp"><img src="img/brands-1.png" alt="Envato">
                            </picture>
                        </a>
                    </div>
                    <div class="brand__block">
                        <a href="">
                            <picture>
                                <source srcset="img/brands-2.jpg" type="image/webp"><img src="img/brands-2.jpg" alt="WordPress">
                            </picture>
                        </a>
                    </div>
                    <div class="brand__block">
                        <a href="">
                            <picture>
                                <source srcset="img/brands-3.webp" type="image/webp"><img src="img/brands-3.png" alt="Tuts+">
                            </picture>
                        </a>
                    </div>
                    <div class="brand__block">
                        <a href="">
                            <picture>
                                <source srcset="img/brands-4.webp" type="image/webp"><img src="img/brands-4.jpg" alt="Microlancer">
                            </picture>
                        </a>
                    </div>
                </div>
            </div>
        </div>

        <section id="contact" class="contact">
            <div class="container">
                <div class="title__page">
                    <h2 class="title title-white">Contact us</h2>
                    <h3 class="sub__title sub__title-white">Proin iaculis purus consequat sem </h3>
                </div>

                <form action="#" method="POST">
                    <div class="contact__form-input">
                        <div class="contact__form">
                            <input class="contact-input " name="name" placeholder="your Name *" required type="text">
                            <input class="contact-input " name="email" placeholder="your e-mail *" required type="email">
                            <input class="contact-input " name="name" placeholder="subject" type="text">
                        </div>

                        <div class="contact__form">
                            <textarea class="contact-input input-message" name="message" placeholder="your Message *" required></textarea>
                        </div>
                    </div>
                    <div class="contact__button">
                        <button type="submit" class="button btn-yelow btn-contact">Send Message</button>
                    </div>
                </form>
            </div>
        </section>

    </main>

    <footer class="footer">
        <div class="container">
            <div class="footer__inner">
                <div class="footer__copyright">
                    Â© Copyright 2014 FreebiesXpress.com
                </div>
                <div class="social footer-social">
                    <a class="twitter-social" href="#"><i class="fa social-block twitter-block  fa-twitter" aria-hidden="true"></i></a>

                    <a class="facebook-social" href="#"><i class="fa social-block facebook-block fa-facebook" aria-hidden="true"></i></a>

                    <a class="google-social" href="#"><i class="fa social-block google-block fa-google-plus" aria-hidden="true"></i></a>
                </div>
            </div>
        </div>
    </footer>

    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
    <script src="https://use.fontawesome.com/9704fdc858.js"></script>
    <script src="js/scrollSmooth.js"></script>
    <script src="js/scroll.js"></script>
    <script src="js/script.min.js"></script>
</body>

</html>